frontend/index.html

```diff
@@ -81,11 +82,11 @@
 
     <div id="maintenanceMode" style='height: 100%; display: none; background-color:rgb(12,47,84); color: white; overflow-y: hidden;'>
       <div style='width:100%; height:100%; margin:0px; text-align: center;'>
-        <img style="margin: 10vh;" src="images/3drepo-logo-white.png">
+        <img style="margin: 10vh;" src="images/3drepo-logo-white.png" longdesc="3DRepoBuildingInformationModellingSoftware" alt="3D Repo">
```

Comment:
I notice you changed some longdesc to have spaces. Should this one get spaces?

Reply:
This has been changed but is no longer needed as the test files are redundant, however i have updated them so they are in line with what is live.